boards/FSM/Src/App/states/App_AllStates.c

```diff
@@ -35,9 +35,9 @@ void App_AllStatesRunOnTick100Hz(struct StateMachine *const state_machine)
     }
 
     // Check for torque plausibility
-    float left_torque_req = App_CanMsgs_dcm_invl_command_message_torque_command_invl_decode(
+    float left_torque_req = (float)App_CanMsgs_dcm_invl_command_message_torque_command_invl_decode(
```

Comment:
not relevant to PR scope

Reply:
Oh yeah, can we group this in? Builds fail without this change for me
(I suspect its because we use different compilers for Linux vs. Windows)